CONTRIBUTING.md

```diff
@@ -185,6 +185,15 @@ use [pre-commit](https://pre-commit.com/) hooks:
 (env) $ make install
 ```
 
+For Windows User:
+```
+# install chocolatey and run "choco install make" to use make functions
+(env) $ cd drf-user
+(env) $ pip install -r requirements-dev.txt
+(env) $ pip install -e .
+pre-commit install
+```
+
```

Comment:
@HardipinderS If you can put these steps in a different section about `setting up in windows system` that would be much better. Please try to put these details in separate section.

Reply:
Added

Reply:
Added the new changes separately for Linux and Windows